pages/common/pdfjoin.md

```diff
@@ -1,15 +1,20 @@
 # pdfjoin
 
-> PDF merging utility.
+> PDF merging utility based on pdfjam.
+> More information: <https://github.com/rrthomas/pdfjam-extras>.
 
-- Merge two PDFs:
+- Merge two PDFs into one with the default suffix "joined":
 
-`pdfjoin {{file1}} {{file2}} --outfile {{output_file}}`
+`pdfjoin {{path/to/file1.pdf}} {{path/to/file2.pdf}}`
 
-- Save pages 3 to 5 followed by page 1 to a new PDF:
+- Merge the first page of each given file together:
 
-`pdfjoin {{file 3-5,1}} --outfile {{output_file}}`
+`pdfjoin {{path/to/file1.pdf path/to/file2.pdf ...}} 1 --outfile {{output_file}}`
```

Comment:
```suggestion
`pdfjoin {{path/to/file1.pdf path/to/file2.pdf ... 1}} --outfile {{output_file}}`
```
What I've seen from the other examples it should be this.

Reply:
What example are you thinking of @marchersimon? That syntax confuses a bit, actually.

Reply:
Well, the syntax is `pdfjoin {{file_paths ...}} {{page_number}}`, which equals `pdfjoin {{file_paths ... page_number}}` and that's what I changed it to. Before it was `pdfjoin {{file_paths ...}} page_number` .